### ISSUE TITLE
In there any way to use this after RNW removed className?

### ISSUE THREAD

**Issue by kuasha420** [kristerkari/react-native-css-modules#37]
I couldn't find anything to explain that.

**Comment by kristerkari** [comment]
I think that with RNW the only possibility is to use something like `patch-package` to patch the library to bring className support back.

**Comment by kristerkari** [comment]
I'm using a patch in the example app that brings back the support for react-native-web `className` prop:
https://github.com/kristerkari/react-native-css-modules-with-typescript-example/blob/master/patches/react-native-web%2B0.12.3.patch

**Comment by aaronyee-eth** [comment]
> I'm using a patch in the example app that brings back the support for react-native-web `className` prop: https://github.com/kristerkari/react-native-css-modules-with-typescript-example/blob/master/patches/react-native-web%2B0.12.3.patch

Hey bro, I was going through some cool css solutions and I still believe that a powerful css-in-file could have the best css feature coverage so that I can utilize expo to fully work on Web without losing any consistency across all platforms.

I appreciate your amazing work on the whole solution!

However, I got quite confused when adding the support to RNW. Is there any possibility that you could fork the RNW and the community can help you to maintain the className support and sync with the upstream?

**Comment by aaronyee-eth** [comment]
I created a fork and modified the package to make it support `className`.

https://github.com/0xGravityLabs/react-native-web

https://www.npmjs.com/package/@gravitylabs/react-native-web

Feel free to use it.

**Comment by aaronyee-eth** [comment]
> > Is there any possibility that you could fork the RNW and the community can help you to maintain the className support and sync with the upstream?
> 
> I don't want to start maintaining a fork of RNW, so that's why I have not done it.
> 
> > I created a fork and modified the package to make it support `className`.
> 
> Nice! Do you also have plans to keep on updating the fork with changes from RNW? Your fork could be added to the README if that's the case.

Yes! I think the most maintaining jobs to do are simply just syncing with the upstream as I didn't change much of the code. I'll try to keep it update-to-date at every major release of RNW.

**Comment by JoeBoulton** [comment]
How do I get this to work in my project? When I uninstall react-native-web and install your fork, expo says it can't find react-native-web. Any tips?

**Comment by kristerkari** [comment]
@JoeBoulton I have not tested, so I'm not sure if it works, but could that be possible with the `alias` feature of Yarn or npm? 

So if you tried to use `react-native-web` as an alias for `@gravitylabs/react-native-web`. 

See: https://www.nieknijland.nl/blog/use-multiple-versions-of-an-npm-package-at-the-same-time
